tests/unit/components/trending-videos-item-test.js

```diff
@@ -40,26 +40,4 @@ module('Unit | Component | trending videos item', (hooks) => {
 
 		assert.equal(componentMock.get('thumbUrl'), `http://vignette/image.jpg/top-crop/${imageWidth}/${imageHeight}`);
 	});
-
-
-	test('handles openLightbox action properly', function (assert) {
```

Comment:
I can see a ticket to fix tests
I think we should also write new ones

Reply:
👍